modules/data.land/R/soilgrids_soc_extraction.R

```diff
@@ -0,0 +1,149 @@
+###Extraction of total soil organic carbon based on user-defined site location from SoilGrids250m version 2.0 : https://soilgrids.org
+###Input parameter: user-defined site longitude/latitude 
+###Extract variable ocd (organic carbon density)
+###Example:ocs<-ocs_extract(c(-77.904, -119.1944, -121.557), c(40.6658, 37.0678, 44.4524))
+
+rm(list = ls())
+library(terra)
+library(reshape2)
+library(dplyr)
```

Comment:
For the final version of the function we put into data.land, we will need to populate the header with documentation.  you can look at other pecan functions to see how to do this.

You will also need to remove the calls to loading these libraries and make sure the dependent functions are part of the DESCRIPTION and NAMESPACE

Reply:
Clarification: the NAMESPACE file should not be edited manually. After you document your functions using ROxygen syntax you should run `make document` and commit the updated version of the NAMESPACE and any Rd documentation files that are generated.

Reply:
To keep things moving I am going to post up here now a quick edit with docs etc to show you what we mean. We will need to still update the code to accept a dataframe of siteID, site name, lat, long, for the function. if we dont use BETY we can just show how site ID can be generic or NA perhaps but if we are using BETY sites that should be included as well as the site name